src/network.c

```diff
@@ -1426,12 +1433,7 @@ static int parse_packet (sockent_t *se, /* {{{ */
 			status = parse_part_encr_aes256 (se,
 					&buffer, &buffer_size, flags);
 			if (status != 0)
-			{
-				ERROR ("network plugin: Decrypting AES256 "
-						"part failed "
-						"with status %i.", status);
```

Comment:
Please keep this message around. It'll mean some duplicate log entries but it ensures that we'll get *some* error message even if someone forgets to log the actual error in `parse_part_encr_aes256` when adding further checks in the future.